legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/api/GitLabProjectConfigurationApi.java

```diff
@@ -401,6 +405,40 @@ public List<MetamodelDependency> getMetamodelDependencies()
         return ProjectStructure.getProjectStructure(projectConfiguration).getAvailableGenerationConfigurations();
     }
 
+    @Override
+    public ProjectConfigurationStatusReport checkProjectConfigurationStatus(String projectId)
+    {
+        return new ProjectConfigurationStatusReport()
+        {
+            @Override
+            public boolean isProjectConfigured()
+            {
+                return getProjectConfiguration(projectId) != null;
+            }
+
+            @Override
+            public String getReviewId()
+            {
+                try
```

Comment:
If isProjectConfigured is true, do you still want to find project configuration reviews?

Reply:
No if project is configured we don't want find the reviews. I will just modify my logic to accomodate that. Thanks!